.circleci/config.yml

```diff
@@ -174,23 +358,25 @@ jobs:
         - persist_to_workspace:
            root: cmake-docker-build-debug
            paths: [ bin_install ]
+           
 
 workflows:
-  build-and-release:
+  version: 2
+  build-and-deploy-image:
+    jobs:
+      - build-and-push-docker-image:
+         context: taraxa-node
+                           
+  create-builder-image:
     jobs:
       - create-and-push-builder:
           context: taraxa-node
           filters:
             branches: 
               only: 
-                #- develop
+#               - develop
```

Comment:
The `master` and `develop` branches will be added to another PR?
asking because of the PR description

Reply:
That's a different job, it is for building the binaries.

Reply:
ops hahah